src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
`any` sounds good but I would rather display the `current` page with a flash msg/warning on top that this may not be the version they are using. In most cases changes aren't going to be so critical between releases that having an intermediate page is beneficial IMO.